de.metas.acct.base/src/main/sql/postgresql/ddl/functions/acctBalanceToDate.sql

```diff
@@ -15,89 +15,93 @@ CREATE TYPE de_metas_acct.BalanceAmt AS
 );
 
  */
-create or replace function de_metas_acct.acctBalanceToDate(p_Account_ID numeric, p_C_AcctSchema_ID numeric, p_DateAcct date, p_AD_Org_ID numeric(10,0),  p_IncludePostingTypeStatistical char(1) = 'N',  p_ExcludePostingTypeYearEnd char(1) = 'N')
-RETURNS de_metas_acct.BalanceAmt AS
+CREATE OR REPLACE FUNCTION de_metas_acct.acctBalanceToDate_test(p_Account_ID numeric,
```

Comment:
cannot follow what's changed here. u will have to tell me....

Reply:
I have refactored the `create or replace` line, so that each param is shown on different lines. This will help with future diffing.

I have also extracted all the joins in the 3 subqueries into a CTE, which brings a speedup of about 30%.

we can skype about this for more details.